doc/usage/installation.rst

```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
 __ https://github.com/sphinx-doc/sphinx
 __ https://github.com/sphinx-doc/sphinx/archive/master.tar.gz
 __ https://github.com/sphinx-doc/sphinx/archive/master.zip
+
+
+docker-sphinx
+-------------
+
+`docker-sphinx <https://hub.docker.com/r/sphinxdoc/sphinx>`_ is Docker images for Sphinx.
+
+It includes images
+
+- sphinxdoc/sphinx
+- sphinxdoc/sphinx-latexpdf
+
+.. note::
+
+   sphinxdoc/sphinx-latexpdf contains TeXLive packages. So the image is very large (over 2GB!).
+
+You can create a Sphinx project::
```

Comment:
The description after this line is not about "installation". So it might be better to move other place.

Reply:
Your right. I added TODO as a tentative solution.